README.md

```diff
@@ -1,17 +1,10 @@
 # Baby Steps with Rust - UDP Datagram
 
-Hey there! I'm Tim. I primarily work as a Ruby on Rails developer, but I've been excited by Rust for a long time. And I have dabbled. Oh, how I have dabbled. But recently I've felt like it was time to write some more substantial Rust code, so I came up with some babysteps...
-
-Any this is the first babystep! .
-
```

Comment:
As with nearly every first draft (my own included!) the first paragraph can be safely removed. Get straight into it!

Reply:
Fair thee well former first paragraph!